managers/bosh-manager/DirectorService.js

```diff
@@ -778,7 +768,13 @@ class DirectorService extends BaseDirectorService {
     return Promise.join(
         this.executeActions(CONST.SERVICE_LIFE_CYCLE.PRE_BIND, actionContext),
         this.getDeploymentIps(deploymentName),
-        (preBindResponse, ips) => this.agent.createCredentials(ips, binding.parameters, preBindResponse)
+        (preBindResponse, ips) => utils.retry(() => this.agent.createCredentials(ips, binding.parameters, preBindResponse), {
+          maxAttempts: 3
+        })
+        .catch(errors.Timeout, err => {
```

Comment:
why is this error logged specifically? This would have anyways got logged in LN790. Seems unnecessary.